source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveActionSetRenderer.cs

```diff
@@ -13,34 +13,40 @@ public static class AdaptiveActionSetRenderer
     {
         public static FrameworkElement Render(AdaptiveActionSet actionSet, AdaptiveRenderContext context)
         {
-            var outerActionSet = new Grid();
+            Grid outerActionSet = new Grid();
```

Comment:
>Grid [](start = 12, length = 4)

use var

Reply:
VS suggested to use explicit type, and I think its suggestion makes sense in this case as the change make the code clearer

---
In reply to: [284904753](https://github.com/microsoft/AdaptiveCards/pull/2927#discussion_r284904753) [](ancestors = 284904753)

Reply:
I would typically refrain from using var for readability unless the RHS is a super complicated lambda or something that we are assigning to.. In this case though VS (public guidelines) vs one of our recommended coding guiedlines conflict and  in that case i would go with the public recommendation (ie prefer explicit type esp given that it aids in readability in this case and is not a complicated type) 

---
In reply to: [285217421](https://github.com/microsoft/AdaptiveCards/pull/2927#discussion_r285217421) [](ancestors = 285217421,284904753)